khal/controllers.py

```diff
@@ -21,6 +21,8 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
>Without this import, literals are str, not bytestrings. With it, they become unicode strings (unicode).

Somehow the rest of the sentence slipped: You first have to look at the top of the file whether `unicode_literals` are imported to make sense of the literals, which is IMO dangerous while reviewing PRs.

>and forgetting that may cause subtle unexpected error

The opposite does too. Try the following code, with `file` having nonascii characters in it:

    with open('file') as f:
        print(u'output: {}'.format(f.read()))